apps/mobile/src/components/add-wallet/add-wallet-sheet.tsx

```diff
@@ -19,12 +19,12 @@ export function AddWalletSheet({ addWalletSheetRef }: AddWalletSheetBaseProps) {
   const router = useRouter();
   const [optionData, setOptionData] = useState<OptionData | null>(null);
   const createWallet = useCallback(() => {
-    router.navigate(APP_ROUTES.WalletCreateNewWallet);
+    router.navigate(AppRoutes.WalletCreateNewWallet);
```

Comment:
Maybe you can fix the `hooks` lint error now too? Just adding `addWalletSheetRef` to the dependancies?